### PR TITLE
chore: add a note of how to pass todoless action

### PR DESCRIPTION
**Pull request by LashaJini**
### Description

- [Contributor couldn't figure out how to pass `GitHub Bot: TODOLess` action](https://toptal-core.slack.com/archives/CERF5NHT3/p1656539086914019), so we make it a lil bit more clear.

### How to test

- Checking `.github/PULL_REQUEST_TEMPLATE.md` should be enough 

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [ ] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [ ] Self reviewed
- [ ] Covered with tests

**Breaking change**

- [ ] codemod is created and showcased in the changeset
- [ ] test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ Describe the changes and motivations for the pull request.
 - [ ] codemod is created and showcased in the changeset
 - [ ] test alpha package of Picasso in StaffPortal
 
+> When everything is done and ready for review, check all the checkboxes to
+> pass the GitHub Bot: TODOLess action
```
[the remark this answers is not in the document]
Comment (LashaJini) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: It probably still won't be clear how these checks are related to TODOLess action.

[the remark this answers is not in the document]
Comment (LashaJini) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: How about:

> When everything is done and ready for review, check rest of the checkboxes...

[the remark this answers is not in the document]
Comment (dmaklygin) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: okay, let's be more concrete for our dear contributors :)

**Review by dmaklygin — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply .github/PULL_REQUEST_TEMPLATE.md:34]: IMHO, we can rephrase and say something like:
`All development checks should be done and set checked to see "GitHub Bot: TODOLess action" is passing.`, WDYT?

**Review by OndrejTuma — APPROVED** [review]
Nice, this is clear to me 👍

**Review by separatio — APPROVED** [review]
Crystal clear. :slightly_smiling_face:

**Pull request merged**